node/jaeger/src/spans.rs

```diff
@@ -149,6 +163,7 @@ pub enum Stage {
 	AvailabilityRecovery = 6,
 	BitfieldDistribution = 7,
 	ApprovalChecking = 8,
+	ApprovalVoting = 9,
```

Comment:
How's that different from approval checking?

Reply:
This was confusion from my side. Not sure why we can't call it either `ApprovalChecking`, or `ApprovalVoting` everywhere.